lib/python/bailoclient/client.py

```diff
@@ -207,9 +207,6 @@ def download_model_files(
             str: Response status code
         """
 
-        if not self.__allow_exports(deployment_uuid):
-            raise ModelFileExportNotAllowed("Files are not exportable for this model")
```

Comment:
can delete this exception now - it's in utils/exceptions.py. 

will also need to remove the import in this file and the reference to the exception in the docstring